lib/semantics/resolve-names.cc

```diff
@@ -1037,11 +1037,260 @@ class ConstructVisitor : public DeclarationVisitor {
   void PopAssociation();
 };
 
+// Resolve OpenMP construct entities and statement (TODO) entities
+class OmpVisitor : public virtual DeclarationVisitor {
+public:
+  static const parser::Name *GetDesignatorNameIf(
```

Comment:
Function name ending in `IF` is confusing to me. I feel `GetDesignatorNameIfDataRef` could be a better name.
Does anyone else have an opinion?

Reply:
I've changed the name. Thanks.